rasa_sdk/endpoint.py

```diff
@@ -23,6 +24,21 @@
 logger = logging.getLogger(__name__)
 
 
+def create_ssl_context(ssl_certificate, ssl_keyfile, ssl_password):
+    """Create a SSL context if a proper certificate is passed."""
```

Comment:
I'd remove that cause we just check whether anything else than `None` was passed. Imo this check does not ensure if it's a `proper` certificate.
```suggestion
    """Create a SSL context if a certificate is passed."""
```